packages/util/src/bytes.ts

```diff
@@ -1,8 +1,33 @@
 import BN from 'bn.js'
-import { intToBuffer, stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
+import { stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
 import { PrefixedHexString, TransformableToArray, TransformableToBuffer } from './types'
 import { assertIsBuffer, assertIsArray, assertIsHexString } from './helpers'
 
+/**
+ * Converts a `Number` into a hex `String`
+ * @param {Number} i
+ * @return {String}
+ */
+export const intToHex = function (i: number) {
+  if (typeof i !== 'number' || isNaN(i) || i === Infinity || i < 0 || i % 1 !== 0) {
```

Comment:
That last check `i % 1 !== 0` will evaluate to false when `i === 0`, which I don't think is what we want, since we want to allow 0.
What do you think of the following:
 ```suggestion
  if (typeof i !== 'number' || isNaN(i) || i === Infinity || i < 0 || (i !== 0 && i % 1 !== 0)) {
```

Reply:
Actually I take that back, I thought that `0 % 1` returned NaN for some reason.

Reply:
0 % 1 = 0?